entity-framework/core/cli/dotnet.md

```diff
@@ -262,7 +262,7 @@ dotnet ef dbcontext scaffold Name=ConnectionStrings:Blogging Microsoft.EntityFra
 The following example skips scaffolding an `OnConfiguring` method. This can be useful when you want to configure the DbContext outside of the class. For example, ASP.NET Core apps typically configure it in Startup.ConfigureServices.
 
 ```dotnetcli
-dotnet ef dbcontext scaffold "Server=(localdb)\mssqllocaldb;Database=Blogging;User Id=myUsername;Password=myPassword;" Microsoft.EntityFrameworkCore.SqlServer --no-onconfiguring
+dotnet ef dbcontext scaffold "Server=(localdb)\mssqllocaldb;Database=Blogging;User Id=myUsername;Password=--;" Microsoft.EntityFrameworkCore.SqlServer --no-onconfiguring
```

Comment:
Is that still a legal pw?

Reply:
Uh, I'm not sure - but it's the same password you had before: you had `Password=--` and then an additional `;pw` which wasn't part of the password, since `;` is the connection string delimiter. And since `pw` isn't a valid option, the connection string was invalid.

Reply:
> Uh, I'm not sure - but it's the same password you had before: you had `Password=--` and then an additional `;pw` which wasn't part of the password, since `;` is the connection string delimiter. And since `pw` isn't a valid option, the connection string was invalid.

that was my intention. 

I should have added:

In the preceding code, replace `--;pw;` with a valid password.

Reply:
If *** is a valid password you can't use it. You have to use something that can never be valid, even better if it throws exceptions

Reply:
> If *** is a valid password you can't use it. You have to use something that can never be valid, even better if it throws exceptions

Sorry @blowdart, I see you resolved the issue.